ScoutSuite/providers/aws/facade/ec2.py

```diff
@@ -102,12 +102,19 @@ async def get_network_interfaces(self, region: str, vpc: str):
     async def get_volumes(self, region: str):
         try:
             volumes = await AWSFacadeUtils.get_all_pages('ec2', region, self.session, 'describe_volumes', 'Volumes')
-            await get_and_set_concurrently([self._get_and_set_key_manager], volumes, region=region)
+            await get_and_set_concurrently([self._get_and_set_key_manager, self._get_and_set_volume_tags], volumes, region=region)
```

Comment:
Some changes in this file (like this example) don't look like syntax changes. Is this PR mixing some content from other (unrelated) PRs?

Reply:
This is also part of my messup.  I'm working off of a fork, so I will get these two changes reverted.